src/types/UIProtocol.ts

```diff
@@ -15,8 +15,8 @@ export enum ProtocolCommand {
   UNKNOWN = 'unknown',
 }
 
-export type ProtocolRequest = [ProtocolCommand, JsonType];
+export type JsonTemp = JsonType | JsonArray; // until JsonType is fixed
```

Comment:
It's fixed in master, JsonType is now correctly defined (but can't be used to extends interface)